boards/renesas/ek_ra8d1/ek_ra8d1.yaml

```diff
@@ -0,0 +1,11 @@
+identifier: ek_ra8d1
+name: Renesas EK-RA8D1
+type: mcu
+arch: arm
+ram: 1024
+flash: 2048
+toolchain:
+  - gnuarmemb
```

Comment:
we could also add zephyr here, couldn't we?

Reply:
@ydamigos , I added it